public/wp-content/themes/ColoredCow/functions.php

```diff
@@ -2,8 +2,12 @@
 
 if ( ! function_exists( 'cc_scripts' ) ) {
     function cc_scripts() {
-        wp_enqueue_script('cc-bootstrap', get_template_directory_uri().'/dist/lib/js/bootstrap.min.js', array('jquery'), '1.0.0', true);
-        wp_enqueue_script('main', get_template_directory_uri().'/main.js', array('jquery', 'cc-bootstrap'), '1.0.0', true);
+        
+        wp_enqueue_script('cc-google-jquery','https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js');
+        wp_enqueue_script('cc-custom-tether','https://cdnjs.cloudflare.com/ajax/libs/tether/1.4.0/js/tether.min.js');
```

Comment:
What does custom represent here? Would it affect the naming if I remove it from each name?